central/complianceoperator/v2/rules/datastore/datastore_impl_test.go

```diff
@@ -71,6 +71,99 @@ func (s *complianceRuleDataStoreTestSuite) TearDownTest() {
 	s.db.Teardown(s.T())
 }
 
+func (s *complianceRuleDataStoreTestSuite) TestGetControl() {
+	ctx := sac.WithAllAccess(context.Background())
+	exampleRule2 := &storage.ComplianceOperatorRuleV2{
+		Id:   uuid.NewV4().String(),
+		Name: "ocp4-api-server-anonymous-auth",
+		Controls: []*storage.RuleControls{
+			{Standard: "CIS-OCP", Control: "1.1.1"},
+			{Standard: "NERC-CIP", Control: "CIP-003-8 R5.1.1"},
+		},
+	}
+
+	exampleRule1 := &storage.ComplianceOperatorRuleV2{
+		Id:   uuid.NewV4().String(),
+		Name: "ocp4-api-server-admission-control-plugin-namespacelifecycle",
+		Controls: []*storage.RuleControls{
+			{Standard: "CIS-OCP", Control: "1.1.1"},
+			{Standard: "CIS-OCP", Control: "2.2.2"},
+			{Standard: "CIS-OCP", Control: "3.3.3"},
+			{Standard: "NERC-CIP", Control: "CIP-003-8 R5.1.1"},
+			{Standard: "NERC-CIP", Control: "CIP-555-9 R5.5.5"},
+		},
+	}
+
+	testCases := map[string]struct {
+		inputRuleNames []string
+		exampleRules   []*storage.ComplianceOperatorRuleV2
```

Comment:
expectedRule doesn't work here because the intention is to provide this as a fixture, or example data to test on. The expectation can currently be derived from the inputs, so we do not need to specify an additional expected field.
I've renamed the variable to `ruleFixtures` which  fits better.